state/indexer/block/kv/kv.go

```diff
@@ -86,6 +86,25 @@ func (idx *BlockerIndexer) Index(bh types.EventDataNewBlockEvents) error {
 	return batch.WriteSync()
 }
 
+func (idx *BlockerIndexer) Prune(retainHeight int64) {
+	heights, err := idx.Search(context.Background(),
```

Comment:
Do we need to search for the heights before trying out the code in 95? As in-will the deletion not fail anyways if we don't have this height.

Reply:
Done